fastify.js

```diff
@@ -389,6 +389,11 @@ function fastify (options) {
   }
 
   function defaultClientErrorHandler (err, socket) {
+    // In case of a connection reset, the socket has been destroyed and there is nothing that needs to be done.
```

Comment:
Can you please add a link to our issue and Node core issue?

Reply:
Sure, will do.

Reply:
Done